packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -50,16 +57,17 @@ class CodeSizeBodyState extends State<CodeSizeBody>
   TreemapNode root;
 
   TabController _tabController;
-  static const Key snapshotTabKey = Key('code size snapshot tab');
-  static const Key diffTabKey = Key('code size diff tab');
+
+  @visibleForTesting
+  static const Key snapshotTabKey = Key('Code Size Snapshot Tab');
+  @visibleForTesting
```

Comment:
I could be wrong, but I expect this will throw a lint when you access this key later to check whether the diff tab is selected